src/elements/filter-menu.html

```diff
@@ -18,6 +18,14 @@
         position: relative;
       }
 
+      :host paper-button {
+        outline: 2px solid transparent;
+      }
+
+      :host paper-button.keyboard-focus {
+        outline-color: unset;
+      }
```

Comment:
This was the cleanest way I could figure out to add a visible focus state to some paper buttons...

Reply:
Yeah, these paper-buttons are definitely a little out of date and don't expose a ton of stuff....this works for now. Ideally paper buttons go away 🙃